src/priyanka/Assignment22/PrimeNumbers.java

```diff
@@ -0,0 +1,35 @@
+package priyanka.Assignment22;
+
+import java.util.Scanner;
+
+/*Assignment-22 : 18th Nov'2020 
+Print all prime numbers between 2 to 100.
+*/
+public class PrimeNumbers {
+
+	void primeNo(int startNum, int endNum) {
+
+		for (int index = startNum; index <= endNum; index++) {
+			int count = 0;
+			for (int innerIndex = 1; innerIndex <= endNum; innerIndex++) {
```

Comment:
innerIndex should start from 2 right,
we cannot identify 1 as prime or non-prime
this is correction is require

Reply:
Hi Harsh, 
I have updated the code, please review.